php/admin-template.php

```diff
@@ -281,10 +281,12 @@ function wp_seo_the_title_character_count( $count ) {
 	?>
 	<p>
 		<?php esc_html_e( 'Title character count: ', 'wp-seo' ); ?>
-		<span class="title-character-count"></span>
-		<?php /* translators: %d: title character count */ ?>
-		<noscript><?php echo esc_html( sprintf( __( '%d (save changes to update)', 'wp-seo' ), $count ) ); ?></noscript>
+		<span class="title-character-count"><?php echo esc_html( $count ); ?></span>
```

Comment:
The value could be a sentence from `wp_seo_get_the_display_character_count()`, though. Do you think I should check `is_numeric()` and escaping accordingly?